hilti/toolchain/src/compiler/jit.cc

```diff
@@ -538,6 +563,9 @@ Result<Nothing> JIT::JobRunner::_waitForJobs() {
                 _jobs.erase(id);
             }
         }
+
+        if ( _jobs_pending.empty() && _jobs.empty() )
```

Comment:
Can this become the condition of the while loop (instead of `true`). Could then also use that same check at the beginning of the method.